docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_popup.htm

```diff
@@ -222,36 +222,38 @@ <h2>Popup Events</h2>
 </table>
 <p>To use popup events, use the following code:</p>
 
-<pre class="prettyprint"><pre class="prettyprint lang-html" style="border:0px; margin:0px">
-&lt;!--Popup HTML code--&gt;
+<p>HTML code:</p>
+<pre class="prettyprint">
 &lt;div id=&quot;popup&quot; class=&quot;ui-popup&quot;&gt;
    &lt;div class=&quot;ui-popup-header&quot;&gt;&lt;/div&gt;
    &lt;div class=&quot;ui-popup-content&quot;&gt;&lt;/div&gt;
-&lt;/div&gt;</pre><pre class="prettyprint lang-js" style="border:0px; margin:0px">/* JavaScript code */
-&lt;script&gt;
+&lt;/div&gt;</pre>
+<p>JS code:</p>
+<pre class="prettyprint">
    /* Use popup events */
    var popup = document.getElementById(&quot;popup&quot;);
    popup.addEventListener(&quot;popupbeforecreate&quot;, function(ev)
    {
       /* Implement code for popupbeforecreate event */
    });
-&lt;/script&gt;
 </pre></pre>
```

Comment:
Would you please check this \</pre>\</pre>?
It looks like it should be \</pre> to me.

Reply:
Fixed